cookbooks/gathering_contexts_when_using_multiple_contexts.rst

```diff
@@ -1,11 +1,12 @@
-Accessing contexts from each other
-==================================
+Gathering Contexts when using Multiple Contexts
+===============================================
 
 When splitting the definitions in multiple contexts, it might be useful to
 access a context from another one. This is particularly useful when migrating
 from Behat 2.x to replace subcontexts.
 
-Behat allows to access the environment in :doc:`hooks </guides/3.hooks>`,
+Behat allows to access the environment in
+:ref:`user-guide--feature-contexts--hooking-into-the-test-process--hooks`,
```

Comment:
you should keep the explicit text for the link, otherwise the text will be ``Hooks`` with a capital letter